<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>239</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton">
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PauseButton">
       <property name="text">
        <string>PAUSE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>40</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>MAX threads:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="MaxThreadSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>MAX Urls:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="MaxUrlCountSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>164</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="URLEdit">
       <property name="text">
        <string>http://www.example.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>215</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Text to search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TestToSearchEdit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>test</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>641</width>
      <height>411</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
